hack/vault-init.sh

```diff
@@ -3,15 +3,16 @@
 #set -x
 set -e
 
-NAMESPACE=${NAMESPACE:-spi-system}
+VAULT_KUBE_CONFIG=${VAULT_KUBE_CONFIG:-$HOME/.kube/config}
```

Comment:
I think
```
VAULT_KUBE_CONFIG=${VAULT_KUBE_CONFIG:-${KUBECONFIG:-$HOME/.kube/config}}
```
would work better so that `$KUBECONFIG` is honored if no special kube config is neede for vault deployment.